### PR TITLE
Adding Data Lake Store mref from build 694

### PR DESCRIPTION
**Pull request by v-alje**

### CODE REVIEW

**Comment by openpublishbuild** [comment]
### :x: Validation status: errors


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[]() | :x:Error | | [Details](#user-content-)

<a name=""></a>
### []()
 - **[Error]** Run script failed with no detailed information. Please retry building your content again. If the issue still happens, open a ticket in http://SiteHelp and include the error report for our team to troubleshoot

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C10%5C5%5C14458122-e42f-10e3-bd3d-ee1c7494cfbf%5CPullRequest%5C201710051903487423-323%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=qfgOjKH9b8XRZDODiwcqTVg0JIkoLzPh1EYUOVIc47g%3D&st=2017-10-05T19%3A03%3A59Z&se=2017-10-06T19%3A13%3A59Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by openpublishbuild** [comment]
### :x: Validation status: errors


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[]() | :x:Error | | [Details](#user-content-)

<a name=""></a>
### []()
 - **[Error]** Run script failed with no detailed information. Please retry building your content again. If the issue still happens, open a ticket in http://SiteHelp and include the error report for our team to troubleshoot

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C10%5C5%5C14458122-e42f-10e3-bd3d-ee1c7494cfbf%5CPullRequest%5C201710051915266330-323%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=9gecsbXg2JFbLw9f7iNryZlj8R5RUpE0I3fYZUZJ3IQ%3D&st=2017-10-05T19%3A16%3A22Z&se=2017-10-06T19%3A26%3A22Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by openpublishbuild** [comment]
### :x: Validation status: errors


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[]() | :x:Error | | [Details](#user-content-)

<a name=""></a>
### []()
 - **[Error]** Run script failed with no detailed information. Please retry building your content again. If the issue still happens, open a ticket in http://SiteHelp and include the error report for our team to troubleshoot

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C10%5C5%5C14458122-e42f-10e3-bd3d-ee1c7494cfbf%5CPullRequest%5C201710052044253767-323%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=cT8aVjTp6qmHEsQ1Ok1QNzgQTiryHojT%2BFRcNS1fqfY%3D&st=2017-10-05T20%3A45%3A25Z&se=2017-10-06T20%3A55%3A25Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: .openpublishing.publish.config.json [hunk 1]
```diff
@@ -54,6 +54,7 @@
         "azurecosmosdbcfp-1.1.0",
         "azurecosmosdbcfp-1.1.1",
         "azurecosmosdbcore-1.5.0",
+        "azuredatalakestorenet-0.1.3-beta"
```
Comment (herohua) [inline .openpublishing.publish.config.json]: @v-alje this line is missing an ending ','

**Comment by openpublishbuild** [comment]
### :x: Validation status: errors


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[]() | :x:Error | | [Details](#user-content-)

<a name=""></a>
### []()
 - **[Error]** Run script failed with no detailed information. Please retry building your content again. If the issue still happens, open a ticket in http://SiteHelp and include the error report for our team to troubleshoot

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C10%5C9%5C14458122-e42f-10e3-bd3d-ee1c7494cfbf%5CPullRequest%5C201710092310086429-323%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=pXGLh1suNTvBTZP4TnZlm8szQBpqYKaFyLSMrHq7pMQ%3D&st=2017-10-09T23%3A08%3A37Z&se=2017-11-09T23%3A13%3A37Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by openpublishbuild** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[.openpublishing.publish.config.json](https://github.com/Azure/azure-docs-sdk-dotnet/blob/Build694/.openpublishing.publish.config.json) | :white_check_mark:Succeeded | |
[xml/FrameworksIndex/azure-dotnet.xml](https://github.com/Azure/azure-docs-sdk-dotnet/blob/Build694/xml/FrameworksIndex/azure-dotnet.xml) | :white_check_mark:Succeeded | |
[xml/FrameworksIndex/azuredatalakestorenet-0.1.3-beta.xml](https://github.com/Azure/azure-docs-sdk-dotnet/blob/Build694/xml/FrameworksIndex/azuredatalakestorenet-0.1.3-beta.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Azure.Batch/ODATADetailLevel.xml](https://github.com/Azure/azure-docs-sdk-dotnet/blob/Build694/xml/Microsoft.Azure.Batch/ODATADetailLevel.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Azure.DataLake.Store.Acl/AclAction.xml](https://github.com/Azure/azure-docs-sdk-dotnet/blob/Build694/xml/Microsoft.Azure.DataLake.Store.Acl/AclAction.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Azure.DataLake.Store.Acl/AclActionExtension.xml](https://github.com/Azure/azure-docs-sdk-dotnet/blob/Build694/xml/Microsoft.Azure.DataLake.Store.Acl/AclActionExtension.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Azure.DataLake.Store.Acl/AclEntry.xml](https://github.com/Azure/azure-docs-sdk-dotnet/blob/Build694/xml/Microsoft.Azure.DataLake.Store.Acl/AclEntry.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Azure.DataLake.Store.Acl/AclScope.xml](https://github.com/Azure/azure-docs-sdk-dotnet/blob/Build694/xml/Microsoft.Azure.DataLake.Store.Acl/AclScope.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Azure.DataLake.Store.Acl/AclStatus.xml](https://github.com/Azure/azure-docs-sdk-dotnet/blob/Build694/xml/Microsoft.Azure.DataLake.Store.Acl/AclStatus.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Azure.DataLake.Store.Acl/AclType.xml](https://github.com/Azure/azure-docs-sdk-dotnet/blob/Build694/xml/Microsoft.Azure.DataLake.Store.Acl/AclType.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Azure.DataLake.Store.RetryPolicies/ExponentialRetryPolicy.xml](https://github.com/Azure/azure-docs-sdk-dotnet/blob/Build694/xml/Microsoft.Azure.DataLake.Store.RetryPolicies/ExponentialRetryPolicy.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Azure.DataLake.Store.RetryPolicies/NoRetryPolicy.xml](https://github.com/Azure/azure-docs-sdk-dotnet/blob/Build694/xml/Microsoft.Azure.DataLake.Store.RetryPolicies/NoRetryPolicy.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Azure.DataLake.Store.RetryPolicies/RetryPolicy.xml](https://github.com/Azure/azure-docs-sdk-dotnet/blob/Build694/xml/Microsoft.Azure.DataLake.Store.RetryPolicies/RetryPolicy.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Azure.DataLake.Store/AdlsClient.xml](https://github.com/Azure/azure-docs-sdk-dotnet/blob/Build694/xml/Microsoft.Azure.DataLake.Store/AdlsClient.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Azure.DataLake.Store/AdlsException.xml](https://github.com/Azure/azure-docs-sdk-dotnet/blob/Build694/xml/Microsoft.Azure.DataLake.Store/AdlsException.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Azure.DataLake.Store/AdlsInputStream.xml](https://github.com/Azure/azure-docs-sdk-dotnet/blob/Build694/xml/Microsoft.Azure.DataLake.Store/AdlsInputStream.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Azure.DataLake.Store/AdlsOutputStream.xml](https://github.com/Azure/azure-docs-sdk-dotnet/blob/Build694/xml/Microsoft.Azure.DataLake.Store/AdlsOutputStream.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Azure.DataLake.Store/ContentSummary.xml](https://github.com/Azure/azure-docs-sdk-dotnet/blob/Build694/xml/Microsoft.Azure.DataLake.Store/ContentSummary.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Azure.DataLake.Store/Core.xml](https://github.com/Azure/azure-docs-sdk-dotnet/blob/Build694/xml/Microsoft.Azure.DataLake.Store/Core.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Azure.DataLake.Store/DirectoryEntry.xml](https://github.com/Azure/azure-docs-sdk-dotnet/blob/Build694/xml/Microsoft.Azure.DataLake.Store/DirectoryEntry.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Azure.DataLake.Store/DirectoryEntryType.xml](https://github.com/Azure/azure-docs-sdk-dotnet/blob/Build694/xml/Microsoft.Azure.DataLake.Store/DirectoryEntryType.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Azure.DataLake.Store/ExpiryOption.xml](https://github.com/Azure/azure-docs-sdk-dotnet/blob/Build694/xml/Microsoft.Azure.DataLake.Store/ExpiryOption.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Azure.DataLake.Store/IfExists.xml](https://github.com/Azure/azure-docs-sdk-dotnet/blob/Build694/xml/Microsoft.Azure.DataLake.Store/IfExists.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Azure.DataLake.Store/LatencyTracker.xml](https://github.com/Azure/azure-docs-sdk-dotnet/blob/Build694/xml/Microsoft.Azure.DataLake.Store/LatencyTracker.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Azure.DataLake.Store/Operation.xml](https://github.com/Azure/azure-docs-sdk-dotnet/blob/Build694/xml/Microsoft.Azure.DataLake.Store/Operation.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Azure.DataLake.Store/OperationCodes.xml](https://github.com/Azure/azure-docs-sdk-dotnet/blob/Build694/xml/Microsoft.Azure.DataLake.Store/OperationCodes.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Azure.DataLake.Store/OperationResponse.xml](https://github.com/Azure/azure-docs-sdk-dotnet/blob/Build694/xml/Microsoft.Azure.DataLake.Store/OperationResponse.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Azure.DataLake.Store/RequestOptions.xml](https://github.com/Azure/azure-docs-sdk-dotnet/blob/Build694/xml/Microsoft.Azure.DataLake.Store/RequestOptions.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Azure.DataLake.Store/SyncFlag.xml](https://github.com/Azure/azure-docs-sdk-dotnet/blob/Build694/xml/Microsoft.Azure.DataLake.Store/SyncFlag.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Azure.DataLake.Store/UserGroupRepresentation.xml](https://github.com/Azure/azure-docs-sdk-dotnet/blob/Build694/xml/Microsoft.Azure.DataLake.Store/UserGroupRepresentation.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Azure.Documents.ChangeFeedProcessor/ChangeFeedEventHost.xml](https://github.com/Azure/azure-docs-sdk-dotnet/blob/Build694/xml/Microsoft.Azure.Documents.ChangeFeedProcessor/ChangeFeedEventHost.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Azure.Documents.Client/AccessCondition.xml](https://github.com/Azure/azure-docs-sdk-dotnet/blob/Build694/xml/Microsoft.Azure.Documents.Client/AccessCondition.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Azure.Documents.Client/ConnectionMode.xml](https://github.com/Azure/azure-docs-sdk-dotnet/blob/Build694/xml/Microsoft.Azure.Documents.Client/ConnectionMode.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Azure.Documents.Client/ConnectionPolicy.xml](https://github.com/Azure/azure-docs-sdk-dotnet/blob/Build694/xml/Microsoft.Azure.Documents.Client/ConnectionPolicy.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Azure.Documents.Client/DocumentClient.xml](https://github.com/Azure/azure-docs-sdk-dotnet/blob/Build694/xml/Microsoft.Azure.Documents.Client/DocumentClient.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Azure.Documents.Client/DocumentResponse`1.xml](https://github.com/Azure/azure-docs-sdk-dotnet/blob/Build694/xml/Microsoft.Azure.Documents.Client/DocumentResponse%601.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Azure.Documents.Client/FeedOptions.xml](https://github.com/Azure/azure-docs-sdk-dotnet/blob/Build694/xml/Microsoft.Azure.Documents.Client/FeedOptions.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Azure.Documents.Client/FeedResponse`1.xml](https://github.com/Azure/azure-docs-sdk-dotnet/blob/Build694/xml/Microsoft.Azure.Documents.Client/FeedResponse%601.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Azure.Documents.Client/IFeedResponse`1.xml](https://github.com/Azure/azure-docs-sdk-dotnet/blob/Build694/xml/Microsoft.Azure.Documents.Client/IFeedResponse%601.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Azure.Documents.Client/Protocol.xml](https://github.com/Azure/azure-docs-sdk-dotnet/blob/Build694/xml/Microsoft.Azure.Documents.Client/Protocol.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Azure.Documents.Client/RequestOptions.xml](https://github.com/Azure/azure-docs-sdk-dotnet/blob/Build694/xml/Microsoft.Azure.Documents.Client/RequestOptions.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Azure.Documents.Client/ResourceResponse`1.xml](https://github.com/Azure/azure-docs-sdk-dotnet/blob/Build694/xml/Microsoft.Azure.Documents.Client/ResourceResponse%601.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Azure.Documents.Client/UriFactory.xml](https://github.com/Azure/azure-docs-sdk-dotnet/blob/Build694/xml/Microsoft.Azure.Documents.Client/UriFactory.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Azure.Documents.Linq/DocumentQueryable.xml](https://github.com/Azure/azure-docs-sdk-dotnet/blob/Build694/xml/Microsoft.Azure.Documents.Linq/DocumentQueryable.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Azure.Documents.Linq/UserDefinedFunctionProvider.xml](https://github.com/Azure/azure-docs-sdk-dotnet/blob/Build694/xml/Microsoft.Azure.Documents.Linq/UserDefinedFunctionProvider.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Azure.Documents.Spatial/GeometryOperationExtensions.xml](https://github.com/Azure/azure-docs-sdk-dotnet/blob/Build694/xml/Microsoft.Azure.Documents.Spatial/GeometryOperationExtensions.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Azure.Documents.Spatial/Polygon.xml](https://github.com/Azure/azure-docs-sdk-dotnet/blob/Build694/xml/Microsoft.Azure.Documents.Spatial/Polygon.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Azure.Documents.SystemFunctions/TypeCheckFunctionsExtensions.xml](https://github.com/Azure/azure-docs-sdk-dotnet/blob/Build694/xml/Microsoft.Azure.Documents.SystemFunctions/TypeCheckFunctionsExtensions.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Azure.Documents/Database.xml](https://github.com/Azure/azure-docs-sdk-dotnet/blob/Build694/xml/Microsoft.Azure.Documents/Database.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Azure.Documents/Document.xml](https://github.com/Azure/azure-docs-sdk-dotnet/blob/Build694/xml/Microsoft.Azure.Documents/Document.xml) | :white_check_mark:Succeeded | |

This comment lists only the first 50 files in the pull request.
For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C10%5C9%5C14458122-e42f-10e3-bd3d-ee1c7494cfbf%5CPullRequest%5C201710092311437828-323%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=jqzRkYXF53Fl%2BiC2BB55IeBKNRIwJj0NCH2MpvtfU%2B4%3D&st=2017-10-09T23%3A45%3A12Z&se=2017-11-09T23%3A50%3A12Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**